Explorer/Assets/Scripts/MVC/Manager/IMVCManager.cs

```diff
@@ -25,6 +25,6 @@ public interface IMVCManager : IDisposable
 
         void SetAllViewsCanvasActive(bool isActive);
 
-        void SetAllViewsCanvasActiveExcept<T>(bool isActive) where T : IController;
+        void SetAllViewsCanvasActive(IController except, bool isActive);

```

Comment:
Wouldn't it be better if we could accept a collection of `IController`?

Reply:
I would go with YAGNI and do it only when need comes